recipes/holopy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "HoloPy" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "fecfcdd3fb46de5b03db7ee53fa9ef6463eb75a1889143578b51707ecaeb878e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/manoharan-lab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k or win or np<112]
+
+requirements:
+  build:
+    - python
```

Comment:
When I add toolchain as a build requirement, f2py compiles Fortran code properly, but throws the following error when trying to compile C:

``In function `_start': (.text+0x20): undefined reference to `main'``

@jakirkham any insight on this? What is toolchain supposed to do?